receiver/kafkareceiver/direct_unmarshaler.go

```diff
@@ -0,0 +1,40 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kafkareceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver"
+
+import (
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+)
+
+type directLogsUnmarshaler struct {
```

Comment:
This naming doesn't represent encoding type as in `direct logs` has no meaning to me.

Considering this is just forcing the logs as a raw message into message body, why not have this as `RawUnmarshaler`?

Reply:
There should also be a Marshaller for this as well.

Reply:
`direct` is used by Apache Camel - I can use `raw` instead.